AdobeStockImageAdminUi/view/adminhtml/web/template/grid/column/preview/related.html

```diff
@@ -5,23 +5,22 @@
  */
 -->
 <div if="isVisible($row())"
-    class="related-container">
+     class="related-container">
     <div id="adobe-stock-tabs"
          class="adobe-stock-tabs"
          data-bind="mageInit: {
                     'mage/backend/tabs': {
-                    active: 'series_content_' + $row().id,
                     destination: '#adobe-stock-tabs-content',
                     shadowTabs: [],
                 }
             }">
         <ul class="tabs-horiz">
-            <li>
+            <li data-bind="if: showSeriesTab($row())">
```

Comment:
I believe this syntax should work
```suggestion
            <li if="showSeriesTab($row())">
```

Reply:
That makes sense. I've been playing around the way that will make the behavior of the tabs more obvious. However, I've ended up with a lot of redundant code.  

Now it should be better. Thanks for the suggestion.